pkg/pushsync/pushsync.go

```diff
@@ -285,8 +289,6 @@ func (ps *PushSync) handler(ctx context.Context, p p2p.Peer, stream p2p.Stream)
 	case errors.Is(err, topology.ErrWantSelf):
 		stored, reason = true, "want self"
 		return store(ctx)
-	case errors.Is(err, ErrShallowReceipt):
-		fallthrough
```

Comment:
Now it will go to default? 
If this is ok, should we have a unit test for this case? both `err==nil` and `default` case are not covered by unit tests...

Reply:
forwarder nodes no longer perform receipt checks, this is why the shallow receipt error check is removed.
see https://github.com/ethersphere/bee/pull/4958/files#diff-522faf538629254cbcb2fed372ba3f9c216497f868f45f6bd82cd6d5d6143c69R471